__tests__/distributors/zulu-installer.test.ts

```diff
@@ -0,0 +1,369 @@
+import fs from 'fs';
+
+import * as tc from '@actions/tool-cache';
+import * as core from '@actions/core';
+import * as ifm from '@actions/http-client/interfaces';
+import { HttpClient } from '@actions/http-client';
+
+import path from 'path';
+import * as semver from 'semver';
+
+import * as utils from '../../src/util';
+import { ZuluDistributor } from '../../src/distributors/zulu/installer';
+import { IZuluVersions } from '../../src/distributors/zulu/models';
+
+let manifestData = require('../data/zulu/zulu-releases-default.json');
+
+describe('getAvailableVersions', () => {
+  let spyHttpClient: jest.SpyInstance;
+  let zuluDistributor: ZuluDistributor;
+  let originalPlatform: NodeJS.Platform;
+
+  beforeEach(() => {
+    spyHttpClient = jest.spyOn(HttpClient.prototype, 'getJson');
+    spyHttpClient.mockImplementation(
+      async (): Promise<ifm.ITypedResponse<IZuluVersions[]>> => {
+        const result = JSON.stringify(manifestData);
+        return {
+          statusCode: 200,
+          headers: {},
+          result: JSON.parse(result) as IZuluVersions[]
+        };
+      }
+    );
+
+    originalPlatform = process.platform;
+    Object.defineProperty(process, 'platform', {
+      value: 'darwin'
+    });
+  });
```

Comment:
Why do we need to override process.platform? Can be just mock `getPlatformOption`?

Reply:
Done. Thank you.